src/appliedsciencestudio/covid19_clj_viz/covid19_in_the_repl.clj

```diff
@@ -21,9 +21,9 @@
 ;; through an Oz server. Oz will open a browser window to display the
 ;; visualizations.
 (comment
-  (oz/start-server! 8082)
```

Comment:
Could you undo these formatting changes? I actually like for `comment` blocks to have a couple blank lines before their end-parenthesis. It makes it easier to evaluate as I move my cursor.